templates/docker-compose.yml.j2

```diff
@@ -50,6 +50,7 @@ services:
       --topics='{{ nim_waku_topics | join(" ") }}'
 {% if "store" in nim_waku_protocols_enabled %}
       --store-message-db-url='sqlite:///data/store.sqlite3'
+      --store-message-db-vacuum=true
```

Comment:
We might want to parametrize this with a variable. For now lets just set it to `false`. Then we can try enabling it on case-by-case basis on different fleets.

Reply:
Variable added

Reply:
Agreed with default `false` here, but good to have available as variable.